be/src/exec/es/es_scroll_parser.cpp

```diff
@@ -306,6 +315,9 @@ void ScrollParser::_append_null(Column* column) {
 Status ScrollParser::_append_value_from_json_val(Column* column, const TypeDescriptor& type_desc,
                                                  const rapidjson::Value& col, bool pure_doc_value) {
     LogicalType type = type_desc.type;
+    if (type != TYPE_CHAR && type != TYPE_VARCHAR) {
+        RETURN_NULL_IF_STR_EMPTY(col, column);
```

Comment:
Why do we need to treat `""` as a null value for other none string types?

Reply:
varchar / char type we can't decide it's really empty str or null value. so here. so here we only solve it's null, but can't read case.